src/app/TimedHandler.h

```diff
@@ -83,11 +111,7 @@ class TimedHandler : public Messaging::ExchangeDelegate
         kExpectingFollowingAction, // Expecting write or invoke.
     };
 
-    // Because we have a vtable pointer and mTimeLimit needs to be 8-byte
-    // aligned on ARM, putting mState first here means we fit in 16 bytes on
-    // 32-bit ARM, whereas if we put it second we'd be 24 bytes.
-    // On platforms where either vtable pointers are 8 bytes or 64-bit ints can
-    // be 4-byte-aligned the ordering here does not matter.
```

Comment:
This comment should stay here, since it explains the weird "not biggest to smallest" ordering of the members.

It's possible that it needs to be modified to account for the new member, though.

Reply:
I will add a comment. I don't think it works anymore: an extra pointer means that everything shifts by 4 bytes so no more savings. The entire structure needs to be 8-byte aligned anyway if mTimeLimit is 8-byte aligned.

I will comment as "leave the member with the largest alignment last".

This will help if I get to include the static_smart_ptr support... I will try to import that in since changing reporting engine seems to take a while (I need to make the interface sane without using friend).

Reply:
Updated and pulled in the static-pointer-support to make the alignment actually give savings on arm embedded.

Reply:
Confirmed padding works with something like:

```
pahole --show_private_classes -C TimedHandler out/nrf-nrf5340dk-all-clusters/modules/connectedhomeip/obj/src/app/interaction-model.TimedHandler.cpp.o
```

maybe something like this should be in the comment to figure out how to get it ... however I am reluctant to do so to not lose CI progress.